pkgs/os-specific/linux/nvme-cli/default.nix

```
@@ -35,6 +35,6 @@ stdenv.mkDerivation rec {
     '';
     license = licenses.gpl2Plus;
     platforms = platforms.linux;
-    maintainers = with maintainers; [ primeos tavyc ];
+    maintainers = with maintainers; [ tavyc ];
```

Not that I use the tool a lot, but I now some stuff about NVME:

```suggestion
    maintainers = with maintainers; [ mic92 ];
```

---

Thank you @Mic92! :)